schema/data/prod/form_json/administration-2020.json

```
@@ -185,6 +257,38 @@
           "ui:placeholder": "ex: A1A 1A1"
         }
       }
+    },
+    "primaryContact": {
+      "phone": {
+        "ui:options": {
+          "inputType": "tel"
+        }
+      },
+      "ui:order": [
+        "firstName",
+        "lastName",
+        "position",
+        "organization",
```

we've been using the 'organisation' spelling everywhere else in the database/app rather than 'organization'
```suggestion
        "organisation",
```